dataland-qa-service/src/main/resources/application.properties

```diff
@@ -1,14 +1,34 @@
 springdoc.show-actuator=true
```

Comment:
In the `application.properties` of the backend we still have the following prop:
`dataland-edc-client.baseUrl=http://dataland-edc:9191/api`

We can remove that I think.

Reply:
Removed that along with a few other edc client references. I wonder how they were still there. To be pushed

Reply:
👍